src/main/java/uk/ac/cam/cl/dtg/segue/dos/content/ItemChoice.java

```diff
@@ -42,4 +43,8 @@ public ItemChoice() {
     public void setItems(final List<Item> items) {
         this.items = items;
     }
+
+    public Boolean isAllowSubsetMatch() { return this.allowSubsetMatch; }
+
+    public void setAllowSubsetMatch(final boolean allowSubsetMatch) { this.allowSubsetMatch = allowSubsetMatch; }
```

Comment:
Could you lay this out on three lines? I know IntelliJ will collapse it for you, but it's not consistent with the rest of the codebase (and I don't like it either! :P )

Reply:
In fact, do this for all of them in the DO and DTO?

Reply:
IntelliJ collapses the three lines into one visually, so the rest of the codebase looked like that to me (and personally I like this way of doing it anyway ':D) - I'll change it so that it's consistent.